### DISCUSSION TITLE
Richer listing of recipes with grouping

### DISCUSSION THREAD

**Discussion by gl-yziquel** [casey/just#1970] [Ideas]
Hi.

There are ways to use awk and co. to create listings with groups of recipes for Makefiles. Which allows for a richer format than what `just` currently proposes. See here:

https://www.thapaliya.com/en/writings/well-documented-makefiles/

Any plans to implement that ? Or would that violate some kind of KISS principle ?

**Comment by casey** [comment]
I think this would be reasonable to add. See #1842, a PR which adds this which I've let sit for way too long 😅

**Comment by gl-yziquel** [comment]
OK. Original issue: https://github.com/casey/just/issues/1776

I've built the PR and will toy with it in the future, when I get the opportunity.

Thank you for finding time to work on just in your busy schedule.
